timepiece/templates/timepiece/navigation-management.html

```diff
@@ -1,44 +1,40 @@
 {% load timepiece_tags %}
 
-{# FIXME: Some of those permissions don't exist #}
 {% if perms.entries.can_clock_in or perms.entries.view_entry_summary or perms.entries.view_payroll_summary or perms.contracts.change_entrygroup or user.is_staff %}
     <li class="dropdown">
         <a href="#" class="dropdown-toggle" data-toggle="dropdown">Management <b class="caret"></b></a>
         <ul class="dropdown-menu">
-            {% if perms.entries.view_payroll_summary %}
-                {% if perms.crm.view_backlog %}
+            
+            {% if perms.crm.view_backlog %}
                 <li><a href="{% url 'report_active_projects_burnup_charts' %}">Active Projects Burnup Charts</a></li>
                 <li><a href="{% url 'report_overrun_backlog' %}">Activity Goals Overrun</a></li>
+            {% endif %}
+            {% if perms.crm.view_employee_backlog %}

```

Comment:
do we want this permission to live in the `crm` app instead of the `reports` app?  I think the latter makes more sense.